pkg/declcfg/helpers_test.go

```diff
@@ -0,0 +1,275 @@
+package declcfg
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/pkg/model"
+	"github.com/operator-framework/operator-registry/pkg/property"
+)
+
+func buildValidDeclarativeConfig(includeUnrecognized bool) DeclarativeConfig {
+	a001 := newTestBundle("anakin", "0.0.1",
+		withChannel("light", ""),
+		withChannel("dark", ""),
+	)
+	a010 := newTestBundle("anakin", "0.1.0",
+		withChannel("light", testBundleName("anakin", "0.0.1")),
+		withChannel("dark", testBundleName("anakin", "0.0.1")),
+	)
+	a011 := newTestBundle("anakin", "0.1.1",
+		withChannel("dark", testBundleName("anakin", "0.0.1")),
+		withSkips(testBundleName("anakin", "0.1.0")),
+	)
+	b1 := newTestBundle("boba-fett", "1.0.0",
+		withChannel("mando", ""),
+	)
+	b2 := newTestBundle("boba-fett", "2.0.0",
+		withChannel("mando", testBundleName("boba-fett", "1.0.0")),
+	)
+
+	var others []Meta
+	if includeUnrecognized {
+		others = []Meta{
+			{Schema: "custom.1", Blob: json.RawMessage(`{"schema": "custom.1"}`)},
+			{Schema: "custom.2", Blob: json.RawMessage(`{"schema": "custom.2"}`)},
+			{Schema: "custom.3", Package: "anakin", Blob: json.RawMessage(`{
+				"schema": "custom.3",
+				"package": "anakin",
+				"myField": "foobar"
+			}`)},
+			{Schema: "custom.3", Package: "boba-fett", Blob: json.RawMessage(`{
+				"schema": "custom.3",
+				"package": "boba-fett",
+				"myField": "foobar"
+			}`)},
+		}
+	}
+
+	return DeclarativeConfig{
+		Packages: []Package{
+			newTestPackage("anakin", "dark", svgSmallCircle),
+			newTestPackage("boba-fett", "mando", svgBigCircle),
+		},
+		Bundles: []Bundle{
+			a001, a010, a011,
+			b1, b2,
+		},
+		Others: others,
+	}
+}
+
+type bundleOpt func(*Bundle)
+
+func withChannel(name, replaces string) func(*Bundle) {
+	return func(b *Bundle) {
+		b.Properties = append(b.Properties, property.MustBuildChannel(name, replaces))
+	}
+}
+
+func withSkips(name string) func(*Bundle) {
+	return func(b *Bundle) {
+		b.Properties = append(b.Properties, property.MustBuildSkips(name))
+	}
+}
+
+func newTestBundle(packageName, version string, opts ...bundleOpt) Bundle {
+	csvJson := fmt.Sprintf(`{"kind": "ClusterServiceVersion", "apiVersion": "operators.coreos.com/v1alpha1", "metadata":{"name":%q}}`, testBundleName(packageName, version))
+	b := Bundle{
+		Schema:  schemaBundle,
+		Name:    testBundleName(packageName, version),
+		Package: packageName,
+		Image:   testBundleImage(packageName, version),
+		Properties: []property.Property{
+			property.MustBuildPackage(packageName, version),
```

Comment:
That would result in callers needing to do:
```
property.Must(property.BuildPackage(packageName, version))
```
which is more stutter.

A generic function already exists: `MustBuild(interface{})`, which calls `Build(interface{})`.

The idea of these property-specific functions is that they should never error/panic since they use structs we've already defined (and thus we know don't error during `json.Marshal`), so with that in mind, its not necessary to expose property-specific error functions that return `*Property, error`.

I wouldn't be opposed to just dropping the `Must` prefix from the property-specific builders though.